signature/src/bls_over_bls12381.rs

```diff
@@ -92,15 +92,20 @@ use derivative::Derivative;
 use tagged_base64::{tagged, TaggedBase64, Tb64Error};
 use zeroize::{Zeroize, Zeroizing};
 
-// #[tagged(tag::BLS_SIGNING_KEY)]
 #[derive(Clone, Derivative, Zeroize)]
 #[zeroize(drop)]
-/// A BLS Secret Key (Signing Key).
+/// A BLS Secret Key (Signing Key). This struct is intentionally made not
+/// serializable so that it won't be unnoticably serialized or printed out
```

Comment:
"not serializable" -> didn't derive serde::Serialize so that outer struct won't automatically derive Serialize and print out ...

Reply:
[33b7fdf](https://github.com/EspressoSystems/jellyfish/pull/696/commits/33b7fdf695340163c966d4962d6fd4db9f6a95f9)